ui/src/components/Entity/EntityHeading.jsx

```diff
@@ -9,6 +10,9 @@ import 'components/common/ItemOverview.scss';
 class EntityHeading extends React.PureComponent {
   render() {
     const { entity, isProfile = false } = this.props;
+    const lastViewedDate = entity.lastViewed ? new Date(parseInt(entity.lastViewed, 10)) : Date.now();
```

Comment:
I'm a bit meh on this, mainly because although it's logical I know that the Date.now() part of this will never actually be used. Stops problems with entity.lastViewed not existing though, which is nice